apps/builder/app/shared/instance-utils.test.ts

```diff
@@ -515,32 +517,227 @@ test("insert template data with only new style sources", () => {
 });
 
 describe("reparent instance", () => {
-  test("from collectioo item", () => {
+  test("between instances", () => {
+    // body
+    //   box
+    //     text
+    //   button
```

Comment:
seems like this comment is missing last text node

Reply:
I usually write only hierarchy of instances. Same like we do in navigator.